web/js/link/wv.link.ui.js

```diff
@@ -14,65 +14,63 @@ wv.link = wv.link || {};
 
 wv.link.ui = wv.link.ui || function(models, config) {
 
-    var self = {};
-    var id = "wv-link-button";
-    var selector = "#" + id;
-    var widgetFactory = React.createFactory(WVC.Share);
+  var self = {};
+  var id = "wv-link-button";
+  var selector = "#" + id;
+  var widgetFactory = React.createFactory(WVC.Share);
 
-    var init = function() {
-      var getLink;
-      var Widget;
+  var init = function() {
+    var getLink;
+    var Widget;
 
-      $button = $("<input></input>")
-          .attr("type", "checkbox")
-          .attr("id", "wv-link-button-check");
-      $label = $("<label></label>")
-          .attr("for", "wv-link-button-check")
-          .attr("title", "Share this map");
-      var $icon = $("<i></i>")
-          .addClass("fa")
-          .addClass("fa-share-square-o")
-          .addClass("fa-2x");
-      $label.append($icon);
-      $(selector).append($label);
-      $(selector).append($button);
-      $button.button({
-          text: false
-      }).click(function() {
-          var checked = $("#wv-link-button-check").prop("checked");
-          WVC.GA.event('Link', 'Click', 'Share link Button');
-          if ( checked ) {
-              self.show();
-          } else {
-              wv.ui.closeDialog();
-          }
-      });
+    $button = $("<input></input>")
+      .attr("type", "checkbox")
+      .attr("id", "wv-link-button-check");
+    $label = $("<label></label>")
+      .attr("for", "wv-link-button-check")
+      .attr("title", "Share this map");
+    var $icon = $("<i></i>")
+      .addClass("fa fa-share-square-o fa-2x");
+    $label.append($icon);
+    $(selector).append($label);
+    $(selector).append($button);
+    $button.button({
+      text: false
+    }).click(function() {
+      var checked = $("#wv-link-button-check").prop("checked");
+      WVC.GA.event('Link', 'Click', 'Share link Button');
+      if ( checked ) {
+        self.show();
+      } else {
+        wv.ui.closeDialog();
+      }
+    });
 
-      models.link.events.on("update", replaceHistoryState);
-    };
+    models.link.events.on("update", replaceHistoryState);
+  };
 
-    self.show = function() {
-      Widget = self.initWidget();
-      self.reactComponent = ReactDOM.render(Widget, $('#share-modal')[0]);
-    };
+  self.show = function() {
+    Widget = self.initWidget();
+    self.reactComponent = ReactDOM.render(Widget, $('#share-modal')[0]);
+  };
 
-    self.initWidget = function() {
-        getLink = models.link.get();
-        return widgetFactory({
-            configs: config,
-            models: models,
-        });
-    };
+  self.initWidget = function() {
+    getLink = models.link.get();
+    return widgetFactory({
+      configs: config,
+      models: models,
```

Comment:
Alright, that makes sense. I'm actually going to close this pull request for the time being too since this is still going through a major rewrite.